src/openrct2/network/network.cpp

```diff
@@ -1136,7 +1142,12 @@ void Network::Server_Send_TICK()
     // but debug version can check more often.
     static sint32 checksum_counter = 0;
     checksum_counter++;
+#ifdef DEBUG_DESYNC 
+    // Check every tick in debug version.
+    if (checksum_counter >= 1) {
+#else
```

Comment:
I've changed it in https://github.com/OpenRCT2/OpenRCT2/pull/5621/commits/a5573aada98aab5a318f361eb10c3ca08a62aac6